salesforce/latest/using/community.md

```diff
@@ -0,0 +1,135 @@
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Salesforce Classic](#share-content-externally-to-community-members-salesforce-classic)
+
+## Share content externally to community members Salesforce Classic
+
+1. Log in to Salesforce.
+
+2. Click **Alfresco Repository** and log in as a community administrator to Alfresco Share in the Canvas app.
+
+      ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
+
+3. Click **Cases** and click the **Create New** drop down list and then select **Case**.
+
+      ![classic-create-case]({% link salesforce/images/classic-create-case.png %})
+
+4. Select the required **Case Origin** and any other information needed for the case. Click **Save**.
+
+      ![classic-new-case]({% link salesforce/images/classic-new-case.png %})
+
+5. Click the **Sharing** button above the case.
+
+      ![classic-sharing]({% link salesforce/images/classic-sharing.png %})
+
+6. Click the **Add** button.
+
+      ![classic-sharing-add]({% link salesforce/images/classic-sharing-add.png %})
+
+7. Search **Public Groups** and add **All Customer Portal Users** to the **Share With** column. Click **Save**.
+
+      ![classic-sharing-groups]({% link salesforce/images/classic-sharing-groups.png %})
+
+8. Click the **Create** drop down list and select **Folder**.
+
+      ![classic-create-folder]({% link salesforce/images/classic-create-folder.png %})
+
+9. Enter a name for the folder and click **Create**.
+
+      ![classic-name-folder]({% link salesforce/images/classic-name-folder.png %})
+
+10. Click the **Actions** button and select **Share Externally**.
+
+      ![classic-share-externally]({% link salesforce/images/classic-share-externally.png %})
+
+11. Select the **Salesforce Community Name** you would like to share with and the **Users**. Click **Save**.
+
+      ![lightning-externally-select]({% link salesforce/images/lightning-externally-select.png %})
+
+You will see a **Successfully Shared Accounts Externally** message. The result of the performed actions is that when the community member logs into Salesforce they will see the case in their **All Open Cases** list.
+
+## Retrieving shared cases
+
+1. Log in to Salesforce Community.
+
+2. Click the **My Open Cases** drop down list and then select **All Open Cases**.
+
+      ![lightning-open-cases]({% link salesforce/images/lightning-open-cases.png %})
+
+3. Click the link of the case number that has been shared with you to open it.
+
+      ![lightning-all-new]({% link salesforce/images/lightning-all-cases.png %})
+
+4. Log in as a Community member to access the case.
+
+      ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
```

Comment:
At the end of this section add: 

Community members can download and view documents linked to the case. They can also upload documents to the case that will get stored in Alfresco.

Reply:
done